src/firebolt/async_db/connection.py

```diff
@@ -285,17 +286,19 @@ def __init__(
         database: str,
         auth: Auth,
         api_endpoint: str = DEFAULT_API_URL,
+        connector_versions: Union[Tuple[str, str], List[Tuple[str, str]]] = None,
```

Comment:
We usually don't create a `Connection` directly, but rather using the `connect` function. Can we add a parameter there as well?
Speaking of having too many parameters there, we could think of just adding an `additional_parameters: dict` argument, where we could later put anything, not used often

Reply:
You're right, I missed this part.